python/plugins/processing/algs/qgis/ExtentFromLayer.py

```diff
@@ -74,6 +77,10 @@ def __init__(self):
 
     def initAlgorithm(self, config=None):
         self.addParameter(QgsProcessingParameterMapLayer(self.INPUT, self.tr('Input layer')))
+        self.addParameter(QgsProcessingParameterNumber(self.ROUND_TO,
+                                                       self.tr('Round values to'), minValue=0,
```

Comment:
Might be worth stating something about "map units" here?

Reply:
I cannot find any other algorithm where units are mentioned. These will just be the same units as used for the bounding box itself